SyntheticsRunTestsTask/__tests__/fixtures.ts

```diff
@@ -37,10 +37,10 @@ const runMockedTask = (mockName: string): MockTestRunner => {
   // See `16.15.0` in `.node-version`
   const nodeVersion = 16
 
-  const task = new MockTestRunner(file)
-  task.run(nodeVersion)
+  const task = await new MockTestRunner().LoadAsync(file)
+  await task.runAsync(nodeVersion)
 
-  // Warnings usually come from `mockery`, and can be useful to spot mocking issues.
```

Comment:
They removed `mockery` (https://github.com/microsoft/azure-pipelines-task-lib/pull/968), and `Replacing existing mock for module` can still be logged, so I updated the comment.